main.js

```diff
@@ -0,0 +1,15 @@
+function ready() {
+  var buttons = Array.from(
```

Comment:
`Array.from` is ES6 - so if we use that we can probs go all-out with `const` and arrow functions? 😝

Reply:
Looks like [no IE support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility)... soooo arrow functions everywhere :P

Reply:
man IE sucks

Reply:
Awww true 🤦‍♂️